app/controllers/Setup.scala

```diff
@@ -149,13 +149,13 @@ object Setup extends LilaController with TheftPrevention {
     PostRateLimit(HTTPRequest lastRemoteAddress ctx.req) {
       NoPlaybanOrCurrent {
         env.forms.hookConfig flatMap { config =>
```

Comment:
you might as well put that `Fu` into the `for` too

Reply:
:+1: